dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -188,3 +191,18 @@
 MSSQL_HOST_PORT = "21433"
 FLOWER_HOST_PORT = "25555"
 REDIS_HOST_PORT = "26379"
+
+EXCLUDE_DOCS_PACKAGE_FOLDER = [
+    'exts',
+    'integration-logos',
+    'rtd-deprecation',
+    '_build',
+    '_doctrees',
+    '_inventory_cache',
+]
+
+
+def get_available_packages() -> List[str]:
```

Comment:
Here, the tests could be added to return list of packages. And compare that the list size is big enough (say > 70 :)  and that it has only components starting of "airflow", "helm-chart", "docker-stack". That would be a nice test.